src/tasks/NetTraceToMibcConverterTask/NetTraceToMibcConverter.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Text.Json.Serialization;
+
+public class NetTraceToMibcConverter : Microsoft.Build.Utilities.Task
+{
+
+    /// <summary>
+    /// Path to the mibc converter tool
+    /// </summary>
+    public string? MibcConverterBinaryPath { get; set; }
+
+    /// <summary>
+    /// Entries for assemblies referenced in a .nettrace file. Important when you run traces against an executable on a different machine / device
+    /// </summary>
+    [Required]
+    public ITaskItem[] Assemblies { get; set; } = Array.Empty<ITaskItem>();
+
+    /// <summary>
+    /// NetTrace file to use when invoking dotnet-pgo for
+    /// </summary>
+    [Required]
+    public string NetTracePath { get; set; } = "";
```

Comment:
Should we extend NetTracePath to be multiple NetTrace files?

Reply:
Does dotnet-pgo support this?

Reply:
No I dont think so, but it does support merging multiple .mibc files. I'm not sure if its a big desire to input multiple .nettrace files, but if it is a use case we could generate multiple .mibc and merge it, or perhaps dotnet-pgo could be extended to convert multiple .nettrace files.